app/Controller/PagesController.php

```diff
@@ -36,6 +37,49 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
+	public $uses3 = array();
+
+/**
+ * TooManyMethods
+*/
+    public function foo(  ){
+
+    }
```

Comment:
__[code_sniffer]__
* Expected 0 blank lines before closing function brace; 1 found
* Function closing brace must go on the next line following the body; found 1 blank lines before brace

<!-- Comment by [SideCI](https://sideci.com) -->